xmtp_proto/src/gen/xmtp.message_api.v1.tonic.rs

```diff
@@ -249,7 +249,7 @@ pub mod message_api_server {
             request: tonic::Request<super::PublishRequest>,
         ) -> std::result::Result<tonic::Response<super::PublishResponse>, tonic::Status>;
         /// Server streaming response type for the Subscribe method.
-        type SubscribeStream: futures_core::Stream<
+        type SubscribeStream: tonic::codegen::tokio_stream::Stream<
```

Comment:
This here, not compiling, with the error that `tokio_stream` can't be found in `tonic::codegen::*` was the tell that something wasn't adding up with the versions.

tonic moved from `futures` to `tokio_stream` for `Stream` in `0.10.0`